### PR TITLE
Add possibility to show or hide ssid

### PR DESCRIPTION
**Pull request by matthieup240**

### CODE REVIEW

File: hassio-hostapd/run.sh [hunk 1]
```diff
@@ -104,6 +105,15 @@ echo "channel=${CHANNEL}" >> ${HCONFIG}
 echo "interface=${INTERFACE}" >> ${HCONFIG}
 echo "" >> ${HCONFIG}
 
+if test ${HIDE_SSID} = true;
+then
+    echo "Hidding SSID"
+    sed -i 's/ignore_broadcast_ssid=0/ignore_broadcast_ssid=1/' ${HCONFIG}
```
Comment (joaofl) [inline hassio-hostapd/run.sh:111]: Why is `sed` required?
Cant you just `echo` it in like the others?

File: hassio-hostapd/run.sh [hunk 2]
```diff
@@ -104,6 +105,15 @@ echo "channel=${CHANNEL}" >> ${HCONFIG}
...
+else
+    echo "Showing SSID"
+    sed -i 's/ignore_broadcast_ssid=1/ignore_broadcast_ssid=0/' ${HCONFIG}
```
Comment (joaofl) [inline hassio-hostapd/run.sh:114]: Same here. You can echo either `ignore_broadcast_ssid=1` or `ignore_broadcast_ssid=0`

**Review by joaofl — COMMENTED** [review]
Nice initiative, i like it. I can see that you are still working on things, so I'll wait before I review it later on.

File: hassio-hostapd/run.sh (hunk 1 above)
Comment (matthieup240) [reply hassio-hostapd/run.sh:111]: `sed` cause the `ignore_broadcast_ssid=0` already exist in the hostapd.conf

**Comment by matthieup240** [comment]
Thanks for the review, it's done.

**Comment by joaofl** [comment]
It looks good! Thanks.
Have you tested it? I am unable to do it right now, since my setup is down at the time.

**Comment by matthieup240** [comment]
I have tested it and works great !

**Pull request merged**